lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+  attr_reader :dishes
+
+  def initialize(menu)
+    @dishes = {}
+    @menu = menu
+  end
+
+  def add(dish, quantity)
+    fail NoItemError, "#{dish.capitalize} is not on the menu!" unless menu.has_dish?(dish)
```

Comment:
Great to check if item exists before adding to order! I totally didn't think of this.